workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/PartnerOpportunitiesList/hooks/useGetListItemsFromPartnerOpportunities.tsx

```diff
@@ -6,25 +6,22 @@
 import {useMemo} from 'react';
 
 import {PartnerOpportunitiesColumnKey} from '../../../common/enums/partnerOpportunitiesColumnKey';
-import useGetDealRegistration from '../../../common/services/liferay/object/deal-registration/useGetDealRegistration';
+import DealRegistrationDTO from '../../../common/interfaces/dto/dealRegistrationDTO';
+import {LiferayAPIs} from '../../../common/services/liferay/common/enums/apis';
+import LiferayItems from '../../../common/services/liferay/common/interfaces/liferayItems';
 import {ResourceName} from '../../../common/services/liferay/object/enum/resourceName';
+import useGet from '../../../common/services/liferay/object/useGet';
 import getOpportunityAmount from '../utils/getOpportunityAmount';
 import getOpportunityDates from '../utils/getOpportunityDates';
 
 export default function useGetListItemsFromPartnerOpportunities(
 	page: number,
 	pageSize: number,
 	filtersTerm: string,
-	sort: string,
-	opportunityFilter?: string
```

Comment:
Why `opportunityFilter` was removed?

Reply:
I changed it to `renewalOpportunitiesFilter`